Setup/EavOptionSetup.php

```diff
@@ -257,4 +267,85 @@ private function workaroundIssue1405()
             $this->appState->setAreaCode('adminhtml');
         }
     }
+
+    /**
+     * Retrieve Attribute Set Id By Id or Name
+     *
+     * @param int|string $entityTypeId
+     * @param int|string $setId
+     * @return int
+     * @throws LocalizedException
+     */
+    public function getAttributeSetId($entityTypeId, $setId)
+    {
+        if (!is_numeric($setId)) {
+            $setId = $this->getAttributeSet($entityTypeId, $setId, 'attribute_set_id');
+        }
+        if (!is_numeric($setId)) {
+            throw new LocalizedException(__('Wrong attribute set ID'));
+        }
+
+        return $setId;
+    }
+
+    /**
+     * Retrieve Attribute set data by id or name
+     *
+     * @param int|string $entityTypeId
+     * @param int|string $id
+     * @param string $field
+     * @return mixed
```

Comment:
Maybe `@return int|string|null`? Then it's at least clear it does not return a compound type.